spec/integration/sidekiq_unique_jobs/lock/until_and_while_executing_spec.rb

```diff
@@ -36,7 +36,21 @@
     allow(process_two).to receive(:runtime_lock).and_return(runtime_two)
   end
 
-  it_behaves_like "a lock implementation"
+  it "can be locked" do
+    expect(process_one.lock).to eq(jid_one)
+  end
+
+  context "when process one has locked the job" do
+    before { process_one.lock }
+
+    it "has locked process_one" do
+      expect(process_one).to be_locked
+    end
+
+    it "prevents process_two from locking" do
+      expect(process_two.lock).to eq(nil)
```

Comment:
The problem here is that you actually want to test the `#execute` method, not the lock method. If `process_one` has locked the job should really `process_two` be allowed to execute? I know it is unlikely to happen given the nature of the job but...